smartseq2_single_sample/ss2_single_sample_kco.wdl

```diff
@@ -0,0 +1,165 @@
+  }
+  output {
+    File junction_table = "SJ.out.tab"
+    File output_bam = "Aligned.sortedByCoord.out.bam"
+    File output_bam_trans="Aligned.toTranscriptome.out.bam"
+  }
+  runtime{
+    docker:"humancellatlas/star_dev:v1"
+    memory: "40 GB"
+    disks :"local-disk 100 HDD"
+  }
+}
+task FeatureCountsUniqueMapping {
+  File aligned_bam
+  File gtf
+  String fc_out
+  command{
+    featureCounts -s 0 -t exon -g gene_id -p -B -C -a ${gtf} -o "${fc_out}.gene.counts.txt" ${aligned_bam}
+    featureCounts -s 0 -t exon -g transcript_id -p -B -C -a ${gtf} -o "${fc_out}.transcripts.counts.txt" ${aligned_bam}
+    featureCounts -s 0 -t exon -g exon_id -p -B -C -a ${gtf} -o "${fc_out}.exon.counts.txt" ${aligned_bam}
+  }
+ runtime{
+    docker:"humancellatlas/star_dev:v1"
+    memory: "10 GB"
+    disks :"local-disk 50 HDD"
+  }
+ output {
+    File genes="${fc_out}.gene.counts.txt"
+    File exons="${fc_out}.exon.counts.txt"
+    File trans="${fc_out}.transcripts.counts.txt"
+  }
+}
+task rsem_expression{
+  File trans_aligned_bam
+  File rsemGenome
+  String rsem_out
+  command{
+    tar -zxvf ${rsemGenome}
+    echo "Aligning fastqs and calculating expression"
+    rsem-calculate-expression --bam --paired-end ${trans_aligned_bam} rsem/rsem_trans_index  "${rsem_out}"
+    cut -f 1,4,5 "${rsem_out}.genes.results" >"${rsem_out}.gene.expected_counts"
+  }
+  runtime{
+    docker: "humancellatlas/rsem"
+    memory: "10 GB"
+    disks: "local-disk 100 HDD"
+  }
+  output{
+    File rsem_gene="${rsem_out}.genes.results"
+    File rsem_transc="${rsem_out}.isoforms.results"
+    File rsem_gene_count="${rsem_out}.gene.expected_counts"
+   }
+}
+task CollectAlignmentSummaryMetrics{
+  File aligned_bam
+  File refGenome_fasta
+  String outputfilename
+  command{
+    java -Xmx10g -jar /usr/picard/picard.jar CollectAlignmentSummaryMetrics \
+      VALIDATION_STRINGENCY=SILENT \
+      METRIC_ACCUMULATION_LEVEL=ALL_READS \
+      INPUT=${aligned_bam} \
+      OUTPUT="${outputfilename}" \
+      REFERENCE_SEQUENCE=${refGenome_fasta} \
+      ASSUME_SORTED=true
+  }
+  output{
+    File alignment_metrics ="${outputfilename}"
+  }
+  runtime{
+    docker:"humancellatlas/picard"
+    memory:"10 GB"
+    disks: "local-disk 10 HDD"
+  }
+}
+task CollectRnaSeqMetrics{
+  File aligned_bam
+  File refGenome_fasta
+  File rRNA_interval
+  String outputfilename
+  File refFlat
+  command {
+    java -Xmx10g -jar /usr/picard/picard.jar  CollectRnaSeqMetrics \
+      VALIDATION_STRINGENCY=SILENT \
+      REF_FLAT=${refFlat} \
+      RIBOSOMAL_INTERVALS=${rRNA_interval} \
+      INPUT=${aligned_bam} \
+      OUTPUT="${outputfilename}" \
+      REFERENCE_SEQUENCE=${refGenome_fasta} \
+      ASSUME_SORTED=true \
+      STRAND_SPECIFICITY=NONE
+  }
+  output {
+     File rna_metrics="${outputfilename}"
+  }
+  runtime{
+        docker:"humancellatlas/picard"
+        memory:"10 GB"
+        disks: "local-disk 10 HDD"
+  }
+}
+workflow ss2_rsem_single_sample_kco {
```

Comment:
`Ss2RsemSingleSampleKco`

If this mimics the KCO pipeline is it necessary to include RSEM in the name? Maybe just `Ss2KcoSingleSample` ?